data/src/main/java/org/oppia/android/data/backends/gae/JsonPrefixNetworkInterceptor.kt

```diff
@@ -13,7 +13,8 @@ import javax.inject.Singleton
  * The Interceptor removes XSSI_PREFIX from every response to produce valid Json.
  */
 @Singleton
-class JsonPrefixNetworkInterceptor @Inject constructor() : Interceptor {
+class JsonPrefixNetworkInterceptor
+@Inject constructor(@XssiPrefix private val xssiPrefix: String) : Interceptor {
```

Comment:
```suggestion
class JsonPrefixNetworkInterceptor @Inject constructor(
  @XssiPrefix private val xssiPrefix: String
) : Interceptor {
```

Formatting is a bit off.

Reply:
Done